src/app/app.component.html

```diff
@@ -1 +1,3 @@
 <router-outlet></router-outlet>
+<app-header title="{{ title }}"></app-header>
```

Comment:
we would want to keep the header above router-outlet, as the latter will get replaced with whatever is in the view after navigation is successful.